test/ngAnimate/animateCssSpec.js

```diff
@@ -1234,6 +1234,67 @@ describe("ngAnimate $animateCss", function() {
             $timeout.flush();
           }).not.toThrow();
         }));
+
+        it("should consider a positive options.delay value for the closing timeout",
```

Comment:
It fails with jquery, because the .css implementation is different from jqlite's